Wrath/Druid.lua

```diff
@@ -2941,7 +2941,7 @@ spec:RegisterOptions( {
g5572f8Cu8gpMkwGPbpLNvwmhooi3VJDofC7HOnEJNWTn0KNbofSRK8gywoDkkppno)yMYli49OKSIh24TAQSAqkZm3spHreMO5SpoHGdypHB0Kn8UiNzGr54wo0yWbAhZw8R4aCgEp4C9WYZHz2yK65I2bA3rk7WbwXRisc6PuCnlPj7Xb08G4e81VIslHNqJhGgWbhC(i4IxeK09GbO8jWzia0qGabPmNS8Gd5jz0I5rLecoJUz14jn0nxsbx3(WobE6umMBpijhy2uB0ntAjIZEBSA65O8pHbRloayv)Batr(aZ7LMK9CX89jzbCUbVoq86X81xGEfq8ci5isF0vqAi7C054scqoO6Kcm57c9MloXJygoWlK88Bn7G1scf9JAHwVIbeieTAvC9YEGOZI5hrjCV7a6UwRV7471Q7rqT4OkJ7JMR8wB1dYkNgqBzhUz5hfA6iv)uhUn2e(dBS4SoGS3JYGSuGtYo2ITOg1lWvedanEzc4aOzWXPqigYq9bgXGYw6HeCw5(GA(uiIbyoBeuZg07kRyhZ8MzvjM(57w8EK415h2SlnpNieBRmDQFdbQIiPqwaQGyD2kVv6xwFAWnuuk4zAxmAAX6nlKZGeXlhyhctsWb7y5c5zG55AQtkLcmicvWcTzN7tOiYZy688sArsmE5Pt6WoTs2a440PrdGr9L7NoEYOwoiI3oDsBz1AppWRG5NoDCeklG)ta5rF3jbkyw)HgNVzXSjw33ERoDAXuDw3GiYmxTATeM2yvicRMuDEkLWWOgoRRqvggjDCjxNPNMnHW9M(rdcAlYo75Py8r0Btb)OXUwglPgJHmln8qi1ajidFmOfVEK7mJ2cpm2)aEzCxztABlH5a4bo2I5LLNhiv3hZyzOVJDkIrAYWbIBZbdWklzyeHEysEusEzbpJSOI4AZy7Re5QzciwpI5U7DC62HzkvI2PtQpTR2hwCNwWP(IvdMnIfe5pev02sQzElj(kHB56aG1L0eB5wxVshxywdUaZGZrdUYSOTp5T4HnMaiwfXMfNHiC5llli5x3iWbzS1SBYCTZI3Y0IxYltJdyCbqB1CtnoYuB(H78vtp7zhXOxnov07uvIgi35rCnvJbndn)qe9bfmb(7yh3iDV4Bx6mgFYfE4Zj8d(0xwg)XE8Br7VOZFgAgKle)dkpAFKI)WzlsUtOnlwRZ5udFP2dmvPSXBHEbhSUNVWIo(JUgdNLli0TpQsgeSxQSbvM9WDl6Tzg2aoSXqZAgSNsY7Que4ze9MvMCm5ql(9GvQiisf1VBAojX5Pnafr5es5bAozJN9UKw4SwK1S5azDR4IIRA3JtpJ6Te7c5KnJ()1nr75NTx(rbCVso4ML24IISnivEyxc1tzdjwgoJaMmZqZ2yhHBjB52eZEuzQj1JCwQmtATXODWVLPcCZjoe8yvKA(t5uOIm0QvTRYF7RqSp8sXWUVD59(BpIiS2bl83(19hYju2OQVRkSMlvHSPkwmV6r)T8FXMHoy7G)5B8bZlkG0)VcpMa1ZbUJ(BhvfA9CjCCv4KQqTM9QcpDsqqh(kFTUbKRPzKDYu9MQcxxf(L7RcN2i)rsCUPqDo)SqOEw9g2iJ13mCE)Tg(O(uWu70ozrE3SOkCgxlDAbbWAU2cRCANOLW193k5eYoNaL4M(oSS1KjxuknbYFIJJ2o9qCwWK5T9jtzAKWQy0TYjDM6Jbwx)6PDAAMrQWYR2bAv4vvHwAwL7jSEd)CyC)eP16AnzATVQO1wvUh2iDsjH370oENt7OttJwT9sSMvigJP37KP6XuDedpGr6xUC9zKHkPcmi3pp3azR7yjfbEmtt(lo1K(7t2f8YD3pOt3ejTwQ356qzx9pRcAzG1zsQokPeaH6mbywbVfdFwA2OTMyLA2UtwD1VXLtVzk6Sp2kaIBExzhEEwh8CBle8apZo0Qc)eaCUO2hXfMUtvOoi)Iub3HXMkIrJCsk0zlqR91ztA693jjmP(6AtWAe(n1mCYZDko9w)7iwDmbC24ojLzrLCDUPWsydQb32vCPRy2Bx2Pl61XY1g3PVM8t6Uk1C8)Z8x11HpEhwh76FkpwLjt06ZA4hFXkL8SkSEwif8nvjMOnrPN7Yy0vNnm(VWKhu29)qC)kgSIBTcFSC7pKXNvV4i9xMQw1A1xxKblQuMKMgWARkS5f13XdzMW31IomhCp3x22)B1qXFE6eqxjam17QlUTVPhvhxaE7VJwa6RyoGN8isXilEhDl0FJkTd2O)sLhyUn1bVcJKBS4E76yDllM0J1Wn9xPzQA8mhSxn1Y0CxzVc6YFMninf08fzmQohffyQ(AI4dOALv9wTcL1B8ouzQ18eDWWQxgV6z6euFTjzxbGwqQgkAVyEA79BgceVNiFxxopjpNgqp1RPxny35GfPGh64wdAdBCaSeJR1NKNJ69WtXYYh92z0pU0n1JzeySR7A)XDAwEEcSj9zVb16Y8SzD)TY7oZpv4JzXbn)a3TOJSMGE2VslH)HsJ5pvOgYJ)WsI1H5hU9gsjZwPNAjt5WC1jI3Q(rZwl9IoDO2F3CMx25Xm79r4I1tL33sdKWwS7p9g3MIXIu79Mooqsc703DJhn2FDJvXwO(W7XZQYqfLs9grAOqQvGAdHyyLAqfO9otklEJYFTbvmSW7hkMPkk2mLrD2pT1zWx3JVUvolVhLMLTneRH3DJSf7nGJU7btAjiuwBBMzQL6kUenDFZvPCaTXtw2MZ4Z320bhwpSQeoAZVVZ2(0owNwGJRVT5koSgwxwOtC4zC3UzovAOmYpZAksw)CEro9TnzmMkYVpO9OKT6MBwpK89TnoTHnpoTk9Rs8jKOAvWQxquFBdu7c0hn2j(WlYcuEo08An4vD9EgGW7Uj(UpTS16F6PZBhFO3o2u)PPTvY650HDFtkr(JqRn1TMpeDtmJRVdEtXQ2(K02gqbJZoR(UBrsF2A5olQpCrf44)zMT)N9cvYgqL)2FbV7FJIEHVE))Zp]] )
```

Comment:
Do you want to keep both the previous version and the new one like we discussed so that this can load in live and PTR?

Reply:
I made the change as discussed in Disc